pkg/deploy/os_specific_windows.go

```diff
@@ -0,0 +1,21 @@
+package deploy
+
+import (
+	"strings"
+)
+
+func convertOsFileName(fileName string) string {
+	return strings.ReplaceAll(fileName, "_windows", "")
+}
+
+func osSpecificSkipFiles() map[string]bool {
+	return map[string]bool{
+		"ccm.yaml":             false, // i should figure out what this is
```

Comment:
CCM includes the RBAC rules for our built in cloud provider.  k3s has an embedded cloud provider that's only function is to set and manage the ExternalIP of a node.  You can disable it with --disable-cloud-controller.  TBH the installation of ccm.yaml should be tied to whether --disable-cloud-controller is set. I can probably make that change as I just ran into an issue with it.

Reply:
Thx for the info!